sdk/search/azure-search-documents/azure/search/documents/_service/_skillsets_client.py

```diff
@@ -176,9 +190,11 @@ def create_or_update_skillset(self, name, **kwargs):
 
         """
         kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        access_condition = None
 
         if "skillset" in kwargs:
```

Comment:
I think we need more consistency in `create_or_update` methods across clients. We have 3 different ways of implementing them already :)

Reply:
Can you be more specific?

Reply:
Will drag this discussion off the PR and send a mail about it.